source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
 
-You cannot take further payments from users using this method without creating
-a new payment. This requires users to re-enter their account details.
+- `description`
+- `reference`
+- `return_url`
 
-## Variable payments using Direct Debit
+#### description
 
-You can collect payments of variable amounts at any time by using a variable agreement.
-For example you may want to collect:
+`description` is a human-readable description of the mandate.
 
-* money each time a service is provided
+The description must be no longer than 255 characters, and must not contain URLs.
 
-* payments where the amounts vary according to service usage
+#### reference
 
-* regular payments where you do not know in advance how much each payment will be
+`reference` is the reference number you want to associate with this mandate.
 
-You could also use this functionality to simulate a recurring or
-scheduled Direct Debit payment, by sending an API call on a set schedule.
-Users would be emailed each time a payment is collected against their mandate.
+The reference number does not need to be unique, and must not:
 
-### Variable Direct Debit payment process
+- be longer than 255 characters
+- contain URLs
 
-These steps happen in order.
+The format is variable.
 
-#### 1. GOV.UK Pay creates the agreement
+#### return_url
 
-API call:  `POST /v1/agreements/`
+`return_url` is an HTTPS URL on your site. GOV.UK Pay will direct your user to this url when they've finished setting up the mandate.
 
-Your service must specify:
+The URL should not be JSON-encoded, because backslashes are invalid characters in the API.
 
-* a `return_url` where GOV.UK Pay will direct the user when they have finished setting up the agreement
 
-* the `agreement_type` (`on demand`)
+### 2. Direct the user to GOV.UK Pay
 
-* a `reference` (optional) - your reference for this agreement
+When the API responds, you need to:
 
-The response contains a `next_url`.
+- direct the user to the `next_url` from the API response
+- store the `mandate_id` from the API response, so you can use it to collect payments against the mandate later
 
-#### 2. Your service directs the user to the `next_url`
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-#### 3. The user provides their account details
+### 3. The user provides their bank account details
 
-For example, account number and sort code for Direct Debit.
+When your service redirects the user to `next_url`, they’ll see a **Set up a Direct Debit** page. The user enters their bank account details and selects **Continue**.
 
-GOV.UK Pay makes an API call to our Direct Debit supplier to verify account
+GOV.UK Pay makes an API call to GoCardless to verify the user's account
```

Comment:
When I firt read this, I thought it was referring to creating the mandate with GoCardless rather than just validating the bank account details. Maybe it could be something like: “GOV.UK Pay checks with GoCardless that the user entered details for a real bank account.”

Reply:
at the risk of hair splitting, i wonder if 'validate' is more accurate since GoCardless don't verify that the account details belong to the person - it's just validating that they're real account details.

Reply:
Good points, thanks! I've clarified in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c, including using 'valid'.